pyproject.toml

```diff
@@ -56,7 +56,7 @@ repl = [
     "click>=8.0.0"
 ]
 simulator = [
-    "aiohttp==3.9.0b0"
+    "aiohttp>=3.9.0b0"
```

Comment:
Let please not start hunting beta versions !!!!

I added the b0, but forgot to limit it to 3.12...see my preference as comment in some of the other PR´s

Reply:
I'm fine with switching to 
```aiohttp==3.9.0b0;python_version>='3.12'
aiohttp==3.8.5;python_version<'3.12'
```

However, it seems orthogonal to this PR, which DreamSorcerer has stated is an improvement regardless of `aiohttp` version.
> The change in this PR is correct, for both current and previous versions of aiohttp (as shown in the docs).
> Even in previous versions of aiohttp, you were never running the cleanup steps for the web app.

Reply:
I have no problem approving the code changes!

Reply:
This PR was not a version bump; in fact the opposite (expanding a pin).

Nevertheless, I have expanded the version strategy so `3.8.6` is permitted on Python <3.12.